lib/jnpr/junos/device.py

```diff
@@ -1275,8 +1275,13 @@ def open(self, *vargs, **kvargs):
         self._nc_transform = self.transform
         self._norm_transform = lambda: JXML.normalize_xslt.encode('UTF-8')
 
-        normalize = kvargs.get('normalize', self._normalize)
-        if normalize is True:
+        # normalize argument to open() overrides normalize argument value
+        # to __init__(). Save value to self._normalize where it is used by
+        # normalizeDecorator()
+        normalize = kvargs.get('normalize')
```

Comment:
Why not just do

        self._normalize = kvargs.get('normalize', self._normalize)
        if self._normalize is True:
            self.transform = self._norm_transform